npm-shrinkwrap.json

```diff
@@ -134,9 +134,9 @@
       "resolved": "https://registry.npmjs.org/buffer-crc32/-/buffer-crc32-0.2.13.tgz"
     },
     "caniuse-db": {
-      "version": "1.0.30000793",
+      "version": "1.0.30000795",
       "from": "caniuse-db@>=1.0.30000161 <2.0.0",
-      "resolved": "https://registry.npmjs.org/caniuse-db/-/caniuse-db-1.0.30000793.tgz"
+      "resolved": "https://registry.npmjs.org/caniuse-db/-/caniuse-db-1.0.30000795.tgz"
```

Comment:
Is this related to the slick grid fix?

Reply:
no, this is just the version that npm pulls without a shrinkwrap based on our current package.json (can't wait for the 1.19 vscode merge)